package.json

```diff
@@ -24,7 +24,7 @@
     "broccoli-asset-rev": "^3.0.0",
     "broccoli-test-helper": "^2.0.0",
     "co": "^4.6.0",
-    "ember-cli": "~3.8.1",
+    "ember-cli": "~3.12.0",
```

Comment:
I'm curious why you went with 3.12, vs 3.16 or 3.18.

Reply:
I'm migrating baby steps, in the next PR's we can migrate to 3.18 or above